src/components/home/sections/boardmembers.scss

```diff
@@ -0,0 +1,40 @@
+.main{ 
+    
+    @media only screen and (max-width: 1024px){
+        display: none;
+        } 
+    display: flex; 
+    direction: row; 
+    @media only screen and (max-width: 430px){
+        display: none;
+    }
+.volunteer-content{ 
```

Comment:
Duplicate class name from volunteer.scss

Reply:
For example, on here we are inside the board member scss and not volunteer content. So it's unclear why we are using "volunteer" here.